src/PrestaShopBundle/Form/Admin/Configure/ShopParameters/ProductPreferences/StockType.php

```diff
@@ -74,7 +74,8 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                     'Decrement products in pack only.' => 1,
                     'Decrement both.' => 2,
                 ],
-            ]);
+            ])
+            ->add('oos_show_label_listing_pages', SwitchType::class);
```

Comment:
You should include label and help inside the FormType directly instead of inside the twig template
In recent forms we include everything in the FormType so that the template only needs to call `form_widget(stockForm)`

Maybe this page has not been updated yet but it's better to follow this convention at least for this new field, this way you can simplify the twig template by simply calling `form_row(stockForm.oos_show_label_listing_pages)`

Reply:
Done.